src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = () => {
+  return new Promise((resolve) => {
+    resolve('Promise was resolved!');
+  });
+};
+
+const promise2 = () => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error('Promise was rejected!'));
+    }, 3000);
+  });
+};
+
+const showSuccessMessage = (message) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = message;
+  body.insertAdjacentElement('beforebegin', div);
+};
+
+const showErrorMessage = (message) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message', 'error-message');
+  div.textContent = `Error: ${message}`;
+  body.insertAdjacentElement('beforebegin', div);
+};
+
+logo.addEventListener('click', () => {
+  promise1()
+    .then((message) => {
+      showSuccessMessage(message);
+    })
+    .catch((error) => {
+      showErrorMessage(error.message);
+    });
+
+  promise2()
+    .then((message) => {
+      showSuccessMessage(message);
+    })
+    .catch((error) => {
+      showErrorMessage(error.message);
+    });
+});
```

Comment:
Fix your code like this for pass all tests
```suggestion
const logo = document.querySelector('.logo');
const body = document.querySelector('body');

const promise1 = new Promise((resolve, reject) => {
  logo.addEventListener('click', () => {
    resolve('Promise was resolved!');
  });
});

const promise2 = new Promise((resolve, reject) => {
  setTimeout(() => {
    reject(new Error('Promise was rejected!'));
  }, 3000);
});

const showSuccessMessage = (message) => {
  const div = document.createElement('div');

  div.classList.add('message');
  div.textContent = message;

  body.appendChild(div);
};

const showErrorMessage = (error) => {
  const div = document.createElement('div');

  div.classList.add('message', 'error-message');
  div.textContent = error.message;

  body.appendChild(div);
};

promise1.then(showSuccessMessage).catch(showErrorMessage);

promise2.then(showSuccessMessage).catch(showErrorMessage);
```